wire-library/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireInput.kt

```diff
@@ -159,4 +176,7 @@ internal class WireInput(var configuration: Provider<Configuration>) {
 
   private val File.isJar
     get() = path.endsWith(".jar")
+
+  private val String.mayBeProject: Boolean
+    get() = startsWith(":")
```

Comment:
Does it sound good enough? Could we ever have another thing that a project starting with `:` ?

Reply:
Seems a bit clumsy. Lets go with it for now; maybe make it fancier later if there’s problems adopting this within Square.